state/event_table.go

```diff
@@ -280,7 +289,7 @@ func (t *EventTable) SelectUnknownEventIDs(txn *sqlx.Tx, maybeUnknownEventIDs []
 	// array entries. But I don't think that's guaranteed. Return an (unordered) set
 	// out of paranoia.
 	queryStr := `
-	WITH maybe_unknown_events(event_id) AS (SELECT unnest($1::text[]))
+	WITH maybe_unknown_events(event_id) AS (SELECT UNNEST($1::text[]))
```

Comment:
Sorry, this is goland autoformatting the SQL. I haven't worked out how to stop it from doing this yet.